ratbag_emu/protocol/hidpp20.py

```diff
@@ -239,3 +240,18 @@ def DeviceInformation(self, data, ase, args):
 
             logger.debug(f'getFwInfo({entityIdx}) = {entity}')
             self.protocol_reply(data, reply)
+
+    def BatteryVoltage(self, data, ase, args):
+        # batteryVoltage, batteryStatus = getBatteryInfo()
+        if ase == 0:
+            voltage = random.randint(0, 4500)
+            flags = random.randint(0, 2)
```

Comment:
In the unit test we would just assert that the values are within range, at least until we implement an internal battery state and a `ratbag-emu` api to interface with it.

Reply:
well, no.

please don't. There is strictly *no* good reason to have randoms in the test devices, when we can not predict which values it will take.

If you really want to have non null values, you can always starts at 100%, every call will decrease the value by, let's say 10%, and so you can actually check that the value changes for instance.